src/main/java/stream/flarebot/flarebot/util/Filter.java

```diff
@@ -19,6 +22,12 @@ public FilterReply decide(ILoggingEvent event) {
                 return FilterReply.DENY;
             }
         }
+        String thread = event.getThreadName();
+        for (String prefix : threadBlacklist) {
+            if (thread.startsWith(prefix)) {
+                return FilterReply.DENY;
```

Comment:
You told me to get rid of them? In my last version I still logged them but stopped the errors. The reason I want the errors from the lava pool to not be passed on is because we can can catch them and even if we catch them the errors still log